mcux/drivers/imx/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ zephyr_library_sources_ifdef(CONFIG_UART_MCUX_LPUART	fsl_lpuart.c)
 zephyr_library_sources_ifdef(CONFIG_VIDEO_MCUX_CSI	fsl_csi.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_MCUX_IMX_WDOG    fsl_wdog.c)
 zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_FLEXCAN	fsl_flexcan.c)
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_RTADC	fsl_adc.c)
```

Comment:
I think the config symbol changed to `CONFIG_ADC_MCUX_RTADC`.

Reply:
Change accordind to https://github.com/zephyrproject-rtos/zephyr/pull/34078#discussion_r613277932